src/util/strongPasswordCheck.ts

```diff
@@ -0,0 +1,35 @@
+export default function (password: string) {
+
+
+  const lowercase = password.match(/[a-z]/g);
+  const uppercase = password.match(/[A-Z]/g);
+  const number = password.match(/[0-9]/g);
+  const specialChar = password.match(
+    /[\!\@\#\$\%\^\&\*\)\(\+\=\.\<\>\{\}\[\]\:\;\'\"\|\~\`\_\-]/g);
+
+  //Returnvalue acts as percentage 
+  if (password.length > 4) {
+    if (lowercase && uppercase && number && specialChar) {
+      return 100;
+    }
+
+    if (lowercase && uppercase && number ||
+      lowercase && uppercase && specialChar ||
+      number && specialChar && lowercase ||
+      number && specialChar && uppercase) {
+      return 75;
+    }
```

Comment:
I'm sure there are more concise ways of writing this logic. This function is a great candidate for unit testing, and if there were unit tests for it I would feel very safe to start experimenting with rewriting this logic. Now, I wouldn't recommend it. 😊